platform/ui/src/components/ContextMenu/ContextMenuItemsBuilder.ts

```diff
@@ -0,0 +1,195 @@
+import { Types } from '@ohif/ui';
```

Comment:
I don't understand a line of this file. Maybe since there is no example/doc on how menus can be created/added etc. 

what are `delegating, checkFunction, subMenu`?
what is `history` ?

Please add a proper documentation either as markdown or top of here to give exampel use cases. This is too much even for me

Reply:
Please see the example file findingsContextMenu, as referenced in the documentation.